Frontend-v1-Original/components/ssVotes/ssVotes.tsx

```diff
@@ -249,7 +249,12 @@ export default function ssVotes() {
     );
   };
 
-  const filteredGauges = useMemo(
+  interface Row {
+    gauge: {
+      tbv: number
+    }
+  }
+  const filteredGauges: Row[] = useMemo(
```

Comment:
This type casting is wrong. In production branch filteredGauges inherits Gauge[] type which has gauge.tbv in it as a required field. Moreover, typescript wouldn't allow you to pass this filtered gauges to GaugesTable, because it awaits type Gauge[]:
```
<GaugesTable
          gauges={filteredGauges}
          setParentSliderValues={setVotes}
          defaultVotes={votes}
          token={token}
/>
```

```suggestion
  const filteredGauges = useMemo(
```